### PR TITLE
[DDW-654] Network status screen renaming

### PR DESCRIPTION
**Pull request by DeeJayElly**
This PR changes Network status screen to Daedalus Diagnostics screen and renames components, strings and all the relevant names in the codebase with new Daedalus Diagnostics name.

## Screenshots
Screenshot 2019-06-04 23 27 05

---

## Testing Checklist

- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1559686501018800)

---

## Translations

Japanese translation:
https://docs.google.com/document/d/1jqniHC-VyOq2eU5AOjmMg7lFOxrR-FJJHUADVRhegrA/edit?usp=sharing

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

File: source/main/menus/osx.js [hunk 1]
```diff
@@ -50,10 +50,10 @@ export const osxMenu = (
         },
       },
       {
-        label: translation('daedalus.networkStatus'),
+        label: translation('daedalus.daedalusDiagnostics'),
         accelerator: 'Command+S',
```
[the remark this answers is not in the document]
Comment (DeeJayElly) [reply source/main/menus/osx.js]: Agree @MarcusHurney. Tnx

File: source/renderer/app/stores/index.js [hunk 2]
```diff
@@ -23,7 +23,7 @@ export const storeClasses = {
   addresses: AddressesStore,
   blockConsolidation: BlockConsolidationStore,
   app: AppStore,
-  networkStatus: NetworkStatusStore,
+  daedalusDiagnostics: DaedalusDiagnosticsStore,
```
Comment (nikolaglumac) [inline source/renderer/app/stores/index.js]: @DeeJayElly please revert the name of the store 🙏

File: source/renderer/app/actions/daedalus-diagnostics-actions.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import Action from './lib/Action';
 
 // ======= NETWORK STATUS ACTIONS =======
 
-export default class NetworkStatusActions {
+export default class DaedalusDiagnosticsActions {
```
Comment (nikolaglumac) [inline source/renderer/app/actions/daedalus-diagnostics-actions.js]: @DeeJayElly please revert the name of the actions to `NetworkStatusActions` 🙏

File: source/renderer/app/stores/index.js (hunk 2 above)
Comment (DeeJayElly) [reply source/renderer/app/stores/index.js]: @nikolaglumac done

File: source/renderer/app/actions/daedalus-diagnostics-actions.js (hunk 3 above)
Comment (DeeJayElly) [reply source/renderer/app/actions/daedalus-diagnostics-actions.js]: @nikolaglumac done

**Comment by nikolaglumac** [comment]
Reviewing this one now...

**Comment by a-rukin** [comment]
image
green paddings are 55px right now, should be 64px -> that's why close button seems like it not in the senter of the right space

**Comment by a-rukin** [comment]
zpl.io/boDJqMM
are there any plans to implement scroll like here? 
if these two comments aren't connected to this PR I can approve

**Comment by nikolaglumac** [comment]
@a-rukin here's my response:

> green paddings are 55px right now, should be 64px -> that's why close button seems like it not in the senter of the right space
I will sort this out.

> zpl.io/boDJqMM are there any plans to implement scroll like here?
There are no plans to implement such a scroll.

**Comment by nikolaglumac** [comment]
@a-rukin scrolling fixed on all overlays as well as paddings on the Daedalus Diagnostics screen:

Screenshot 2019-06-06 at 13 18 02

**Pull request merged**